### PR TITLE
Switch cinder csi operator to step registry

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: ci-operator/config/openshift/openstack-cinder-csi-driver-operator/openshift-openstack-cinder-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -51,6 +51,10 @@ tests:
   commands: make verify
   container:
     from: src
+- as: e2e-openstack
+  steps:
+    cluster_profile: openstack
```
Comment (mandre) [inline ci-operator/config/openshift/openstack-cinder-csi-driver-operator/openshift-openstack-cinder-csi-driver-operator-master.yaml:56]: In a follow-up commit you may also want to move this job to vexxhost.

**Comment by pierreprinetti** [comment]
/hold

Prior to this change, the CI executed the "minimal" suite.

By switching to the "openshift-e2e-openstack-minimal" workflow, [the job will not run any test any more](https://github.com/openshift/release/blob/7464cb76d88fb40ffd72728f1b79e4623ba48c75/ci-operator/step-registry/openshift/e2e/openstack/minimal/openshift-e2e-openstack-minimal-workflow.yaml#L3-L7).

In order to stay consistent and run a minimal test suite, I would suggest to run the `experimental/reliability/minimal` suite instead.

Here is an example where we do exactly that: https://github.com/openshift/release/blob/7464cb76d88fb40ffd72728f1b79e4623ba48c75/ci-operator/config/openshift/installer/openshift-installer-master.yaml#L315-L320

**Review by mandre — COMMENTED** [review]
/hold cancel
/lgtm

**Pull request merged**